src/github.com/robertkrimen/otto/.gitignore

```diff
@@ -1,5 +0,0 @@
-/.test
```

Comment:
For future reference, it's nice to put testing code in the _test.go files, which keeps it from getting compiled into the main app.